.github/workflows/coverage.yml

```diff
@@ -0,0 +1,30 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
```

Comment:
Could we mention somewhere in the comment that code coverage info will be included?

Reply:
yes, updated!